sprint1/login/login.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>로그인페이지</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+    <main>
+        <img class="login-logo" src="./images/loginLogo.png" alt="로그인탑이미지">

```

Comment:
❗️ 수정요청
유저들은 일반적으로 페이지에서 로고를 클릭하면 index 페이지로 이동되는 것을 기대합니다. 또 저희 요구사항 중에 로고를 누르면 index로 이동하라는 내용이 있으니 수정해주시면 좋겠습니다~

```suggestion
        <a href="/">
          <img class="login-logo" src="./images/loginLogo.png" alt="판다마켓">
        </a>
```

추가로 img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때 어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.  
따라서 가능한 해당 이미지를 잘 표현할 수 있는 문자열을 넣어주는 것이 좋습니다.